libs/dataset-explorer/src/lib/getDatasetBoxOption.ts

```diff
@@ -25,7 +26,7 @@ export function getDatasetBoxOption(plotlyProps: IPlotlyProperty): any {
         fillColor: FabricStyles.fabricColorPalette[0],
         lineWidth: 35
       },
-      name: "outlier",
+      name: localization.Core.BoxPlot.outlierLabel,
```

Comment:
It seems this code actually isn't in core but in dataset explorer project, going from the file location "[libs/dataset-explorer/src/lib/getDatasetBoxOption.ts](https://github.com/microsoft/responsible-ai-toolbox/pull/1348/files#diff-35195be7508785732abb8a2cc3115f17a4c9fbb56204bd9b811fbee7e3cbf195)".  I think the localization path should be modified to:

localization.DatasetExplorer.BoxPlot.outlierLabel

Reply:
unless this is used in core as well... but I don't see it in core

Reply:
It seemed like one of the few occasions where localization text is actually reusable, but sure, we can make it custom.